### PR TITLE
Add `finetuning-scheduler` to Foundation Model Fine Tuning section

### PR DESCRIPTION
**Pull request by speediedan**
Thanks again for maintaining this useful summary of the LLMOps package landscape! 

This PR humbly suggests the addition of the [`finetuning-scheduler`](https://github.com/speediedan/finetuning-scheduler) package.
image

`finetuning-scheduler` is a PyTorch Lightning extension that accelerates and enhances foundation model experimentation with flexible fine-tuning schedules. 

The popular PyTorch Lightning package uses this extension for its `Fine Tuning` [tutorial](https://lightning.ai/docs/pytorch/stable/tutorials.html). It's primary utility is facilitating foundation model research and exploration with advanced fine-tuning configurations. Full disclosure, I'm the primary maintainer of `finetuning-scheduler` and a frequent contributor to both PyTorch and PyTorch Lightning.

I appreciated you considering this PR. Thanks again for this useful catalog!

### CODE REVIEW

**Review by gaocegege — APPROVED** [review]

**Comment by gaocegege** [comment]
LGTM :+1:
/lgtm

**Comment by gaocegege** [comment]
Thanks for your contribution! :tada: :+1:
